package.json

```diff
@@ -1,18 +1,37 @@
 {
   "name": "stylelint-config-upleveled",
-  "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "0.0.17",
```

Comment:
ok I think v1.0.0 has some problems (broken image, typos, etc) - maybe this can be [unpublished](https://docs.npmjs.com/policies/unpublish) so that we can keep `v1.0.0` as a good first version

I guess this PR needs a few full careful reviews first